### DISCUSSION TITLE
Local storage only? (Android)

### DISCUSSION THREAD

**Discussion by Mein-E** [osmandapp/OsmAnd#19335] [Q&A]
On an Android phone, where are the points that I drop on an offline map stored?

I need to prevent any GeoData related to my use of osmand from being stored in the cloud. I only want them stored on the phone. Is this possible?

**Comment by DmitryAlexei** [comment]
OsmAnd doesn't store your geodata, the app can work completely offline. 
At the same time, your settings and favorites can be store in your Android account. You can disable Auto-backup option in menu > settings > OsmAnd settings > Auto-backup. For more information, see https://osmand.net/docs/user/personal/global-settings#auto-backup.
